permissions/interfaces.any.js

```diff
@@ -11,18 +11,20 @@ if (self.importScripts) {
 // https://w3c.github.io/permissions/#idl-index
 
 promise_test(async () => {
+  const idl = await fetch("/interfaces/permissions.idl").then(r => r.text());
+  const dom = await fetch("/interfaces/dom.idl").then(r => r.text());
```

Comment:
Nit:... these can sometimes contain things that some browsers don't support... like `AbortController`. This means that tests fail, but for the wrong reasons. 

I don't know that the right approach is here...